e2e/test/scenarios/dashboard-filters-2/dashboard-filters-query-stages.cy.spec.ts

```diff
@@ -1955,7 +1835,6 @@ describe("scenarios > dashboard > filters > query stages + temporal unit paramet
 
 describe("pivot tables", () => {
   const QUESTION_PIVOT_INDEX = 0;
-  const QUESTION_PIVOTED_TABLE_INDEX = 1;
```

Comment:
Removing pivoted table from this test, see https://github.com/metabase/metabase/pull/49222/files#r1818974939